packages/dx-react-scheduler/src/plugins/month-view.jsx

```diff
@@ -31,7 +32,7 @@ export class MonthView extends React.PureComponent {
     };
 
     const {
-      viewName, firstDayOfWeek, intervalCount,
+      name: viewName, firstDayOfWeek, intervalCount,
```

Comment:
Why we use the alias name here?

Reply:
To make code more eloquent.

Reply:
It's very strange. We create property and use it with an alias name everywhere. Why we don't create a property with `alias name` at the beginning?

Reply:
In the context of the 'MonthView', the 'view' postfix for the `name` property is redundant. It's already in the component name. Alias is only useful in our internal code. Look at this:

```
this.dayScaleComputed = getters => computed(
  getters, name, this.dayScaleBaseComputed, getters.dayScale,
);
```

What does the `name` mean? You should find the `name` in the code to understand what it is. 

```
this.dayScaleComputed = getters => computed(
  getters, viewName, this.dayScaleBaseComputed, getters.dayScale,
);
```

Any questions about `viewName` meaning? I don't think so))

Reply:
This is a full response, thank you)